statesman-engine/src/main/java/io/appform/statesman/engine/handlebars/HandleBarsHelperRegistry.java

```diff
@@ -354,6 +356,30 @@ private void registerAdd() {
         });
     }
 
+    private void registerComputeDays() {
+        handlebars.registerHelper("computeDays", (String context, Options options) -> {
+            try {
+                if (null == options.params || options.params.length == 0) {
```

Comment:
Add blank string check for `context` as well as `params[0]`

Reply:
done